tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "strict": true,
     "allowSyntheticDefaultImports": false,
     "outDir": "build",
-    "noUnusedLocals": true
+    "noUnusedLocals": true,
+    "noImplicitAny": false
```

Comment:
I would be against this. I know this is painful, but trust me, if you have to use "any" then you want to set it by yourself.

Reply:
I'm working on tests, and this onImplicityAny will be deleted after doing all job :D